translate-pixiv-tags.user.js

```diff
@@ -355,6 +458,24 @@ function safeMatch (string, regex, group = 0, defaultValue = "") {
 
 const safeMatchMemoized = _.memoize(safeMatch, memoizeKey);
```

Comment:
BTW, it used only once, in other places called just `safeMatch`.

Reply:
Yeah, I don't know why I didn't add it in those other place. I'll add a commit that will do that though.